### ISSUE TITLE
Make usable in Fragments and custom ViewGroups

### ISSUE THREAD

**Issue by cxvii** [plattysoft/Leonids#48]
At the moment ParticleSystem requires an Activity to be instantiated. 
The activity is used to retrieve the parent view calling findViewById() and to get a Resources instance.
This makes tricky to use the lib inside a Fragment (you can call getActivity there, but findViewById may fail) and in a custom ViewGroup (no way to get the Activity here, and most likely the ViewGroup is the parent view you need). 
The solution to this problem would be to have another public constructor that takes a ViewGroup and a Resources instead of an Activity and a viewResId, keep the old one for compatibility but make it call the new constructor

**Comment by plattysoft** [comment]
That's an interesting suggestion, thanks!

I have very limited time lately, so I have no idea of when I'll be able to implement it, but I'll put it into the TODO.

**Comment by ryanwr** [comment]
I've been looking to do just this, I've added a constructor that accepts a parent ViewGroup as well as a drawable. I'd be happy to open a pull request in a bit, just cleaning up a bit.

As another note (I should probably open another issue), when placed inside a layout with WRAP_CONTENT it causes the layout to expand to the size of the layouts parent, My current work around is that I've added a method to set LayoutParams on the internal view that's added. However it would be better if the drawing view sets it's size according to the parents view.
